unison-src/transcripts/diff-namespace.output.md

```diff
@@ -319,95 +316,115 @@ shown, only their also-conflicted dependency is shown.
 ```unison
 a = 333
 b = a + 1
+
+forconflicts = 777
 ```
 
 ```ucm
-  ☝️  The namespace .nsx is empty.
-
-.nsx> add
+scratch/nsx> add
 
   ⍟ I've added these definitions:
   
-    a : ##Nat
-    b : ##Nat
+    a            : Nat
+    b            : Nat
+    forconflicts : Nat
 
-.> fork nsx nsy
+scratch/nsx> branch /nsy
 
-  Done.
+  Done. I've created the nsy branch based off of nsx.
+  
+  Tip: To merge your work back into the nsx branch, first
+       `switch /nsx` then `merge /nsy`.
 
-.> fork nsx nsz
+scratch/nsx> branch /nsz
 
-  Done.
+  Done. I've created the nsz branch based off of nsx.
+  
+  Tip: To merge your work back into the nsx branch, first
+       `switch /nsx` then `merge /nsz`.
 
 ```
 ```unison
 a = 444
 ```
 
 ```ucm
-.nsy> update.old
+scratch/nsy> update.old
 
   ⍟ I've updated these names to your new definition:
   
-    a : ##Nat
+    a : Nat
 
 ```
 ```unison
 a = 555
 ```
 
 ```ucm
-.nsz> update.old
+scratch/nsz> update.old
 
   ⍟ I've updated these names to your new definition:
   
-    a : ##Nat
+    a : Nat
 
-.> fork nsy nsw
+scratch/nsy> branch /nsw
 
-  Done.
+  Done. I've created the nsw branch based off of nsy.
+  
+  Tip: To merge your work back into the nsy branch, first
+       `switch /nsy` then `merge /nsw`.
 
-.> debug.alias.term.force nsz.a nsw.a
+scratch/nsw> debug.alias.term.force .forconflicts .a
 
   Done.
 
-.> debug.alias.term.force nsz.b nsw.b
+scratch/nsw> debug.alias.term.force .forconflicts .b
 
   Done.
 
 ```
 ```ucm
-.> diff.namespace nsx nsw
+scratch/main> diff.namespace /nsx: /nsw:
 
   New name conflicts:
   
-    1. a#uiiiv8a86s : Nat
-       ↓
-    2. ┌ a#mdl4vqtu00 : Nat
-    3. └ a#vrs8gtkl2t : Nat
+    1.  a#uiiiv8a86s : Nat
+        ↓
+    2.  ┌ a#mdl4vqtu00 : Nat
+    3.  └ a#r3msrbpp1v : Nat
     
-    4. b#lhigeb1let : Nat
-       ↓
-    5. ┌ b#aapqletas7 : Nat
-    6. └ b#unkqhuu66p : Nat
+    4.  b#lhigeb1let : Nat
+        ↓
+    5.  ┌ b#r3msrbpp1v : Nat
+    6.  └ b#unkqhuu66p : Nat
   
   Added definitions:
   
-    7. patch patch (added 1 updates)
+    7.  patch patch (added 1 updates)
+  
+  Name changes:
+  
+    Original            Changes
+    8.  forconflicts    9.  a#r3msrbpp1v (added)
+                        10. b#r3msrbpp1v (added)
 
-.nsw> view a b
+scratch/nsw> view a
 
-  a#mdl4vqtu00 : ##Nat
+  a#mdl4vqtu00 : Nat
   a#mdl4vqtu00 = 444
   
-  a#vrs8gtkl2t : ##Nat
-  a#vrs8gtkl2t = 555
-  
-  b#aapqletas7 : ##Nat
-  b#aapqletas7 = ##Nat.+ a#vrs8gtkl2t 1
+  a#r3msrbpp1v : Nat
+  a#r3msrbpp1v = 777
```

Comment:
I use the same original value for creating conflicts for both a and b now (`777`) because we can't yet alias terms from one branch into another.